lib/tasks/ahoy.rake

```diff
@@ -0,0 +1,14 @@
+namespace :ahoy do
+  desc "Filling empty geo-location entries in ahoy_visit table"
+
+  task fill_empty_location_entries: :environment do
+    # Gather all visits with nil longitude, this should suffice.
+    visits = Ahoy::Visit.where(:longitude => nil)
```

Comment:
please read in batches, there could be thousands of visits and it could overload the server

https://api.rubyonrails.org/classes/ActiveRecord/Batches.html

Reply:
ok